### PR TITLE
CAMEL-18006. camel-jbang package image command

### PR DESCRIPTION
**Pull request by mgubaidullin**
https://issues.apache.org/jira/browse/CAMEL-18006

### CODE REVIEW

File: dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/Image.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "image", description = "Create Docker and OCI container images")
+public class Image implements Callable<Integer> {
+
+    private static final Logger LOG = LoggerFactory.getLogger(Image.class);
+
+    @CommandLine.Option(names = {"-h", "--help"}, usageHelp = true, description = "Display the help and sub-commands")
+    private boolean helpRequested;
+    @CommandLine.Option(names = {"-f", "--from"}, description = "Base Image", defaultValue = "gcr.io/distroless/java:11")
```
Comment (oscerd) [inline dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/Image.java:43]: Out of curiosity, do we choose the distroless one for specific reason?

**Review by oscerd — APPROVED** [review]
LGTM, just a question.

File: dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/Image.java (hunk 1 above)
Comment (mgubaidullin) [reply dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/Image.java:43]: 1. You could choose your own base image `--from registry.access.redhat.com/ubi8/openjdk-11-runtime:1.11`
2. Distroless creates the smallest image size. It is very useful during development

**Comment by mgubaidullin** [comment]
@davsclaus @oscerd could you please help. 

main pr build passed OK https://github.com/apache/camel/actions/runs/2208968172 
however build failed https://github.com/apache/camel/actions/runs/2209258320

` Failed to execute goal net.revelc.code.formatter:formatter-maven-plugin:2.17.1:validate (validate) on project camel-jbang-core: File '/home/runner/work/camel/camel/dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/Image.java' has not been previously formatted.  Please format file and commit before running validation!`

**Pull request merged**